handler/cache/handler.go

```diff
@@ -110,7 +111,7 @@ func (h *reqHandler) knownRanged() {
 func (h *reqHandler) knownFull() {
 	utils.CopyHeadersWithout(h.obj.Headers, h.resp.Header())
 	h.resp.Header().Set("Content-Length", strconv.FormatUint(h.obj.Size, 10))
-	h.resp.WriteHeader(h.obj.Code)
+	h.resp.WriteHeader(http.StatusOK)
```

Comment:
I am not sure this hould use `http.StatusOK`: some HTTP responses are cacheable, but not 200. For example, [301](https://en.wikipedia.org/wiki/HTTP_301)